src/index.js

```diff
@@ -0,0 +1,15 @@
+import './styles/index.css';
+import logo from './assets/logo.png';
+import { MovieController } from './Controller/MovieController';
```

Comment:
```suggestion
import { MovieController } from './Controller';
```
index.js를 하나 만들어서 export 용도로 사용해주면 어떨까 싶네요!

Reply:
적용하였습니다!